src/gameplay/game.ts

```diff
@@ -1970,6 +1974,18 @@ class Game extends Room {
       return;
     }
 
+    if (!this.gameTimer.isTimerSet()) {
+      if (!leaveTriggered) {
+        socket.emit('messageCommandReturnStr', {
+          message: 'There is no active timer currently.',
+          classStr: 'server-text',
+        });
+        return;
+      } else {
```

Comment:
Can actually get rid of this else and put the return out a scope. 

Then you can remove the return in the `if (!leaveTriggered)` as well